Cargo.lock

```diff
@@ -5538,9 +5538,9 @@ dependencies = [
 
 [[package]]
 name = "tracy-client"
-version = "0.17.1"
+version = "0.17.4"
```

Comment:
Oooh, should this have been done together with one of the later wgpu bumps? :no_mouth:

Reply:
I think at this point we can allow auto updates to Tracy stuff, but cargo-deny dupes